index.html

```diff
@@ -526,6 +526,13 @@
         subscription</a> MUST be <a>deactivated</a>.
       </p>
       <p>
+        The <a>push endpoint</a> MUST NOT enable information about the user to be derived by actors
+        other than the <a>push service</a>, such as the user's device, identity or location.
+        <a>Push services</a> that do not require <a>push subscriptions</a> to be restricted to an
+        <a>application server</a> [[!WEBPUSH-VAPID]] MUST NOT generate predictable
```

Comment:
Why is this requirement only levied on endpoints that are NOT restricted?  RFC 8030 is pretty clear on this point and restriction to an application server doesn't change that.